Games/JV.AD/script.js

```diff
@@ -51,28 +50,60 @@ graphics.endFill();
 app.stage.addChild(graphics);
 
 //labeling sides
+function generateRandom(min = 9, max = 15) {
+
+  // find diff
+  let difference = max - min;
 
-s1 = 10
-s2 = 7
-s3 = 5
-const hyp = new PIXI.Text(s1);
-hyp.x = 130;
-hyp.y = 120;
-app.stage.addChild(hyp);
+  // generate random number 
+  let rand = Math.random();
 
-const obj = new PIXI.Text(s2);
-obj.x = 100;
-obj.y = 260;
-app.stage.addChild(obj);
+  // multiply with difference 
+  rand = Math.floor( rand * difference);
 
-const adj = new PIXI.Text(s3);
-adj.x = 5;
-adj.y = 150;
-app.stage.addChild(adj);
+  // add with min value 
+  rand = rand + min;
+
+  return rand;
+}
+//hypotenuse has to be greater than two other side 
+
+ let s1 = (generateRandom()); 
+ let s2 = Math.ceil(Math.random()*8);
+ let s3 = Math.ceil(Math.random()*8);
+  
+  const hyp = new PIXI.Text(s1);
+  hyp.x = 130;
+  hyp.y = 120;
+  app.stage.addChild(hyp);
+  
+  const obj = new PIXI.Text(s2);
+  obj.x = 100;
+  obj.y = 260;
+  app.stage.addChild(obj);
+  
+  const adj = new PIXI.Text(s3);
+  adj.x = 5;
+  adj.y = 150;
+  app.stage.addChild(adj);
+  
+  const theta = new PIXI.Text('θ');
+  theta.x = 36;
+  theta.y = 72;
+  app.stage.addChild(theta);
+  
+  //insert trig ratio values in buttons 
+  btn1 = document.getElementById("btn1");
+  btn1.innerHTML = `${s2}/${s1}`;
+  
+  btn2 = document.getElementById("btn2");
+  btn2.innerHTML = `${s2}/${s3}`;
+  
+  btn3 = document.getElementById("btn3");
+  btn3.innerHTML = `${s3}/${s1}`;
+  }
```

Comment:
What is this closing bracket for?  I don't see the corresponding oppening bracket.

Reply:
Oh, this is all inside the  `nextQuestion` function.... so you have a function `generateRandom` within this function... unusual as it makes you function very long